docs/conf/eu/2017/schedule.rst

```diff
@@ -0,0 +1,88 @@
+:template: 2017/eu-content.html
+
+
+Schedule
+========
+
+Write the Docs is more than a conference. Each year we organize a wide
+range of events so that people can come together, collaborate, and learn
+from each other in different ways.
+
+Saturday, September 9
+---------------------
+
+The only event scheduled on Saturday is the :doc:`Boat Tour of Prague </conf/eu/2017/boat>`.
+If you get into town early, join us and experience Prague from the water.
+
+Sunday, September 10
+--------------------
+
+Writing Day & Reception will be held at **Auto Klub, Opletalova 1337/29, Praha 1**.
+
+During the day, we'll hold our :doc:`Writing Day documentation
+sprints </conf/eu/2017/writing-day>`, followed by the conference
+reception in the evening.
+
+We encourage everyone to drop by on Sunday evening for the conference
+reception. You'll have a chance to get acquainted with each other over
+some drinks and snacks.
+
+We'll also help groups organize dinner plans, so you can continue your
+conversations over more substantial food as well.
+
+.. datatemplate::
+   :source: /_data/eu-2017-writing-day.json
+   :template: include/schedule.rst
+
+Monday, September 11
+--------------------
+
+The main conference will take place at **Auto Klub, Opletalova 1337/29, Praha 1**.
+
+This is the main event! Hear from lots of interesting folks about all
+things documentation.
+
+Main Stage
+~~~~~~~~~~
+
+.. datatemplate::
+   :source: /_data/eu-2017-day-1.json
+   :template: include/schedule.rst
+
+Unconference Room
```

Comment:
I suspect the headings got a bit muddles here.

# Schedule

## Sat

## Sun

## Mon

### Main

But the unconf is ## again, right? And I expect that will cascade down through the schedule?